tests/test_location.py

```diff
@@ -1,10 +1,91 @@
 """Test cases for the location module."""
+from pytest_mock import MockFixture
 from rdflib import Graph
 from rdflib.compare import graph_diff, isomorphic
+from skolemizer.testutils import skolemization
 
 from datacatalogtordf import Location
 
 
+def test_to_graph_should_return_identifier_set_at_constructor() -> None:
+    """It returns a centroid graph isomorphic to spec."""
+    location = Location("http://example.com/locations/1")
+    location.centroid = "POINT(4.88412 52.37509)"
+
+    src = """
+    @prefix dct: <http://purl.org/dc/terms/> .
+    @prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+    @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+    @prefix dcat: <http://www.w3.org/ns/dcat#> .
+    @prefix locn: <http://www.w3.org/ns/locn#> .
+    @prefix geosparql: <http://www.opengis.net/ont/geosparql#> .
+
+    <http://example.com/locations/1> a dct:Location ;
+        dcat:centroid \"POINT(4.88412 52.37509)\"^^geosparql:asWKT ;
```

Comment:
Er ikke nødvendig med escape

Reply:
Ok, takk for tips @NilsOveTen , dette kode som henger med fra tidligere